controller/k8s/api.go

```diff
@@ -189,6 +193,9 @@ func NewAPI(k8sClient kubernetes.Interface, spClient spclient.Interface, namespa
 		case Svc:
 			api.svc = sharedInformers.Core().V1().Services()
 			api.syncChecks = append(api.syncChecks, api.svc.Informer().HasSynced)
+		case Job:
+			api.job = sharedInformers.Batch().V1().Jobs()
+			api.syncChecks = append(api.syncChecks, api.svc.Informer().HasSynced)
```

Comment:
```golang
api.syncChecks = append(api.syncChecks, api.job.Informer().HasSynced)
```

Reply:
Ah! Sorry for the silly mistake :disappointed: . Will make sure to check from next PR's